The latest kong version (2.3.1) returns an error on GO plugin integration

### Summary

The latest kong version (2.3.1) returns an error on GO plugin integration.  

### Steps To Reproduce

1. Download https://github.com/gszr/kong-demos/tree/master/02-go-plugins 
2. Run the example 
3. OK
4. Replace version on  2.2.1-alpine and run the same example 
5. OK
6. Replace kong version in the docker file to the latest or 2.3.2-alpine  
7. Run the example
8. See an error 

### Additional Details & Logs

- Kong version (2.3.2)
- Kong error logs 
	[C]: in function 'assert'
	/usr/local/share/lua/5.1/kong/init.lua:489: in function 'init'
	init_by_lua:3: in main chunk
nginx: [error] init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:489: error loading plugin schemas: on plugin 'go-hello': go-hello plugin is enabled but not installed;
no plugin found
stack traceback:
	[C]: in function 'assert'
	/usr/local/share/lua/5.1/kong/init.lua:489: in function 'init'
	init_by_lua:3: in main chunk

- Operating system 
 System Version:	macOS 11.2.1 (20D74
Server: Docker Engine - Community
 Engine:
  Version:          20.10.2
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15

@fffonion 
In case of the container is started with disabled plugin:

/usr/local/bin/go-pluginserver flag redefined: kong-prefix
panic: /usr/local/bin/go-pluginserver flag redefined: kong-prefix

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc0000b8120, 0x998d60, 0xc000099f50, 0x7f0812be836f, 0xb, 0x7f0812bfb0e6, 0x4d)
	/usr/local/go/src/flag/flag.go:851 +0x4a5
flag.(*FlagSet).StringVar(...)
	/usr/local/go/src/flag/flag.go:754
flag.(*FlagSet).String(0xc0000b8120, 0x7f0812be836f, 0xb, 0x7f0812be997a, 0xf, 0x7f0812bfb0e6, 0x4d, 0x200000003)
	/usr/local/go/src/flag/flag.go:767 +0xa6
flag.String(0x7f0812be836f, 0xb, 0x7f0812be997a, 0xf, 0x7f0812bfb0e6, 0x4d, 0xb9ff40)
	/usr/local/go/src/flag/flag.go:774 +0x6e
github.com/Kong/go-pdk/server.init()
	/go/src/github.com/Kong/go-pdk/server/server.go:26 +0x66
plugin.open(0xc0000ae120, 0x23, 0x0, 0xed7c1733d, 0xb9fac0)
	/usr/local/go/src/plugin/plugin_dlopen.go:101 +0xc54
plugin.Open(...)
	/usr/local/go/src/plugin/plugin.go:32
main.(*PluginServer).loadPlugin(0xc0000c0190, 0xc0000ae10b, 0x8, 0x0, 0x0, 0x0)
	/go/src/github.com/Kong/go-pluginserver/pluginserver.go:123 +0x21c
main.(*PluginServer).GetPluginInfo(0xc0000c0190, 0xc0000ae10b, 0x8, 0xc000116100, 0x0, 0x0)
	/go/src/github.com/Kong/go-pluginserver/pluginserver.go:239 +0x78
main.dumpAll()
	/go/src/github.com/Kong/go-pluginserver/main.go:81 +0x1ed
main.main()
	/go/src/github.com/Kong/go-pluginserver/main.go:147 +0x7f

--- a/kong/runloop/plugin_servers/process.lua
+++ b/kong/runloop/plugin_servers/process.lua
@@ -56,8 +56,9 @@ local function get_server_defs()
   if not _servers then
     _servers = {}
 
-    if config.pluginserver_names then
+    if config.pluginserver_names[1] then
       for i, name in ipairs(config.pluginserver_names) do
+        name = name:lower()
         kong.log.debug("search config for pluginserver named: ", name)
         local env_prefix = "pluginserver_" .. name:gsub("-", "_")
         _servers[i] = {

